### PR TITLE
Wire up the full ClearlyDefined definition importer from their blobst…

### PR DESCRIPTION
**Pull request by bobmcwhirter**
…ore.

### CODE REVIEW

**Review by jcrossley3 — COMMENTED** [review]
I don't understand why we need both a `ClearlyDefinedLoader` taking a `Value` and a `ClearlyDefinedCurationLoader` taking a `Curation`.

I'm not familiar enough with the walker interfaces, so I'll defer to @ctron on that.

**Comment by bobmcwhirter** [comment]
> I don't understand why we need both a `ClearlyDefinedLoader` taking a `Value` and a `ClearlyDefinedCurationLoader` taking a `Curation`.

The `Curation` YAML was fairly small/easy to make a `serde` impl for.

The `Definition` JSON is more complex, and since we're just plucking a few small bits out of it, figured JsonPath was sufficient against `Value`.

tl;dr: lazy

File: modules/importer/src/runner/clearly_defined/walker.rs [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        }
+    }
+
+    pub fn continuation(mut self, continuation: ClearlyDefinedItemContinuation) -> Self {
+        self.continuation = continuation;
+        self
+    }
+
+    pub async fn run(mut self) -> Result<ClearlyDefinedItemContinuation, Error> {
+        let changes = reqwest::get(self.changes_index_url()).await?;
```
Comment (ctron) [inline modules/importer/src/runner/clearly_defined/walker.rs:47]: This creates a new connection pool for each request. I think we should create and keep a client.

Same for the other calls.

File: modules/ingestor/src/service/sbom/clearly_defined.rs [hunk 2]
```diff
@@ -14,50 +22,165 @@ impl<'g> ClearlyDefinedLoader<'g> {
...
+                document_id: previously_found.document_id,
+                warnings: vec![],
+            });
+        }
 
-        let sbom = self
-            .graph
-            .ingest_sbom(labels, digests, &curation.document_id(), &curation, &tx)
-            .await?;
+        println!("{:#?}", item);
```
Comment (ctron) [inline modules/ingestor/src/service/sbom/clearly_defined.rs:44]: No `println` please.

**Review by ctron — CHANGES_REQUESTED** [review]
A few small things.

**Review by ctron — APPROVED** [review]
